spec/README.md

```diff
@@ -79,18 +79,18 @@ a version 1 app can verify version 1, 2, 3, or 4 proofs. However, a version
 <table>
   <thead>
     <tr>
-      <th rowspan=2>Version</th>
+      <th rowspan=2>App Identity<br />Version</th>
       <th rowspan=2>Nonce</th>
       <th rowspan=2>Digest Algorithm</th>
-      <th colspan=4>Can Verify</th>
+      <th colspan=4>Can Verify Proof From Version</th>
```

Comment:
If we’re doing `<br />` on `App Identity <br> Version`, should we do `Can Verify Proof <br> from Version`?